<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMultisignature</class>
 <widget class="QDialog" name="DlgMultisignature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multisignature Address Interactions</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resource.qrc">
    <normaloff>:/ico/icon.ico</normaloff>:/ico/icon.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="m_twMain">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="m_tabCreate">
      <attribute name="title">
       <string>Create MiltiSignature Address</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="m_sbMinSignatures">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_lMinSignatures">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Enter the minimum number of signatures requires to sign transactions</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="m_lAddress">
           <property name="text">
            <string>Address Label:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_leAddressLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="m_btnAddAddress">
           <property name="text">
            <string>Add Address / Key</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resource.qrc">
             <normaloff>:/png/add.png</normaloff>:/png/add.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="m_lwAddressList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="movement">
          <enum>QListView::Snap</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QPushButton" name="m_btnCreate">
           <property name="text">
            <string>Create</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resource.qrc">
             <normaloff>:/png/save.png</normaloff>:/png/save.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_lStatus">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="m_teStatus">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="m_lInfo">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use bolow to quickly import an address by its redeem. Don`t forget to add a label before clicking import! &lt;br/&gt;Keep in mind, the wallet will rescan the blockchain to find transactions containing the new address. &lt;br/&gt;Please be patient after clicking import.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="m_btnImportRedeem">
           <property name="text">
            <string>Import Redeem</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resource.qrc">
             <normaloff>:/png/receive.png</normaloff>:/png/receive.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_leRedeem">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabCreateTx">
      <attribute name="title">
       <string>Create MultiSignature Tx</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="m_lInputs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Inputs:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnCoinControl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Coin Control</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="m_lQuantity">
             <property name="text">
              <string>Quantity Selected:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="m_lQuantityValue">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>0                                        </string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="m_lAmount">
             <property name="text">
              <string>Amount:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="m_lAmountValue">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnAddRawInput">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add a Raw Input</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="m_lwHashList">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="m_lAddressAmount">
           <property name="text">
            <string>Address / Amount</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnAddDestination">
           <property name="text">
            <string>Add Destination</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resource.qrc">
             <normaloff>:/png/add.png</normaloff>:/png/add.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="m_lwAddressAmount">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="m_btnCreateTx">
           <property name="text">
            <string>Create</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resource.qrc">
             <normaloff>:/png/save.png</normaloff>:/png/save.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_lStatusTx">
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="m_teCreateStatus">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabSignTx">
      <attribute name="title">
       <string>Sign MultiSignature Tx</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="m_lTransationsHex">
           <property name="text">
            <string>Transaction Hex:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_leTransactionHex">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="m_btnSign">
           <property name="text">
            <string>Sign</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resource.qrc">
             <normaloff>:/png/editpaste.png</normaloff>:/png/editpaste.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnCommit">
           <property name="text">
            <string>Commit</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resource.qrc">
             <normaloff>:/png/send.png</normaloff>:/png/send.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnAddPrivateKey">
           <property name="text">
            <string>Add Private Key</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resource.qrc">
             <normaloff>:/png/add.png</normaloff>:/png/add.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_lHint">
           <property name="text">
            <string>Sign with only private keys (Not Decommended)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="m_lStatusSign">
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="m_lwSignKeysList">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
